tests/test_books.py

```diff
@@ -57,3 +57,29 @@ def test_user_profile_page(client, user, user_books, snippet):
     response = client.get(f'/users/{user.username}')
     html = response.content.decode()
     assert snippet in html
+
+
+def test_user_profile_page_no_favorites(client, user):
```

Comment:
yeah let's remove these 2, let's remove the favorites div as it clutters it up a bit, can you also delete the corresponding list variable in the view? thanks

Reply:
done